spec/models/genre_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Genre, type: :model do  
+  describe 'DB table' do
+    it { is_expected.to validate_presence_of :name }
+  end
+
+  describe 'Factory' do
+      before :each do
+      @performer = FactoryBot.create(:performer)
+      
+      it 'should have valid Factory' do
+            expect(create(:genre)).to be_valid
+      end
+    end
```

Comment:
This looks weird. Please format your code to see if your before block is correct, okay?